app/models/agents/twitter_stream_agent.rb

```diff
@@ -221,8 +229,7 @@ def handle_status(status)
         return if status.has_key?('delete')
         return unless status['text']
         status['text'] = status['text'].gsub(/&lt;/, "<").gsub(/&gt;/, ">").gsub(/[\t\n\r]/, '  ')
-
-        if status["retweeted_status"].present? && status["retweeted_status"].is_a?(Hash)
+        if agent.options[:include_retweets] != "true" && status["retweeted_status"].present? && status["retweeted_status"].is_a?(Hash)
```

Comment:
```suggestion
        if options[:include_retweets] != "true" && status["retweeted_status"].present? && status["retweeted_status"].is_a?(Hash)
```

Reply:
This seemed to have introduced additional errors https://travis-ci.org/github/huginn/huginn/jobs/750836640

I believe the function will work within the agent, but I'm not sure how to write the spec so it can test both a default and non-default option after the agent worker has been initialized with the default config.